src/test/java/uk/gov/companieshouse/itemhandler/kafka/EmailSendMessageFactoryTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.companieshouse.itemhandler.kafka;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.kafka.test.context.EmbeddedKafka;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.companieshouse.email.EmailSend;
+import uk.gov.companieshouse.kafka.deserialization.DeserializerFactory;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.serialization.AvroSerializer;
+import uk.gov.companieshouse.kafka.serialization.SerializerFactory;
+
+@SpringBootTest
+@DirtiesContext
+@EmbeddedKafka
+@TestPropertySource(properties={"certificate.order.confirmation.recipient = nobody@nowhere.com",
+        "certified-copy.order.confirmation.recipient = nobody@nowhere.com"})
+public class EmailSendMessageFactoryTest {
```

Comment:
Are both of these properties being used in this test? Are both cases tested?

Reply:
The properties are needed to build test context correctly. The two email addresses have been covered under `EmailServiceIntegrationTest`